app/gateway/3.0.x/observability/logs.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Logging Reference
+---
+
+## Log Levels
+
+Log levels are set in [Kong's configuration](/gateway/{{page.kong_version}}/reference/configuration/#log_level). Following are the log levels in increasing order of their severity: `debug`, `info`,
+`notice`, `warn`, `error` and `crit`.
+
+- *`debug`:* It provides debug information about the plugin's runloop and each individual plugin or other components. Only to be used during debugging since it is too chatty.
```

Comment:
```suggestion
- *`debug`:* It provides debug information about the plugin's run loop and each individual plugin or other components. Only to be used during debugging since it is too chatty.
```